### PR TITLE
Bump pineappl dependency to fix LO bug

### LINKED ISSUE #121: Issue when creating a Leading Order table
I've tried to create a Leading Order version of theory 400. In order to do that I've copied all relevant files and just set PTO: 0.
I'm using the packages that one gets doing `pip install pineko`. I also tried using the latest master of `pineko` just in case with the same result.

```
eko: 0.13.5
pineappl: 0.6.1
```

This works for _many_ fktables but it fails for some of them. I'm going to use as example `ATLASZRAP36PB` but the same problems can be found with other datasets.

In the `lo_failed.zip` file you can find the operator and theory card 
<details>
<summary> log </summary>
Analyze ATLASZRAP36PB
┌───────────────┐
│ Computing ... │
└───────────────┘
   /home/juacrumar/pinekoing/theories_slim/data/grids/390/ATLASWZRAP36PB-ATLAS-arXiv:1109.5141-Z0_eta34.pineappl.lz4
 + /home/juacrumar/pinekoing/theories_slim/data/ekos/390/ATLASWZRAP36PB-ATLAS-arXiv:1109.5141-Z0_eta34.tar
 = /home/juacrumar/pinekoing/theories_slim/data/fktables/390/ATLASWZRAP36PB-ATLAS-arXiv:1109.5141-Z0_eta34.pineappl.lz4
 with max_as=1, max_al=0, xir=1.0, xif=1.0
thread '<unnamed>' panicked at 'Nothing returned from evolution.: EvolutionFailure("no operator for x1 = 0.002038117507440189 found")', pineappl_py/src/grid.rs:542:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Traceback (most recent call last):
  File "/home/juacrumar/pinekoing/pinekovortex/bin/pineko", line 8, in <module>
    sys.exit(command())
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/pineko/cli/theory_.py", line 95, in fks
    ).fks(pdf)
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/pineko/theory.py", line 453, in fks
    self.iterate(self.fk, tcard=tcard, pdf=pdf)
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/pineko/theory.py", line 212, in iterate
    f(name, grid, **kwargs)
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/pineko/theory.py", line 422, in fk
    _grid, _fk, comparison = evolve.evolve_grid(
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/pineko/evolve.py", line 221, in evolve_grid
    fktable = grid.evolve(
  File "/home/juacrumar/pinekoing/pinekovortex/lib/python3.10/site-packages/pineappl/grid.py", line 340, in evolve
    self.raw.evolve(
pyo3_runtime.PanicException: Nothing returned from evolution.: EvolutionFailure("no operator for x1 = 0.002038117507440189 found")
</details>

If I change `PTO: 1` in the theory and rerun the creation of the ekos then it does work. This is in `nlo_ok.zip`. As you can see the only difference between the two theory cards is the PTO. Since it might be possible that other things need to change for the theory to work at LO, I'm also reproducing the theory card below:

```
ID: 390
Comments: NNPDF4.0 NNLO alphas=0.118 with new pipeline
CKM: 0.97428 0.22530 0.003470 0.22520 0.97345 0.041000 0.00862 0.04030 0.999152
DAMP: 0
EScaleVar: 1
FNS: FONLL-C
GF: 1.1663787e-05
HQ: POLE
IC: 0
IB: 0
MP: 0.938
MW: 80.398
MZ: 91.1876
MaxNfAs: 5
MaxNfPdf: 5
ModEv: TRN
ModSV: None
NfFF: 4
PTO: 0
Q0: 0.75
QED: 0
Qmb: 4.92
Qmc: 1.51
Qmt: 172.5
Qref: 91.2
Qedref: 91.2
SIN2TW: 0.23126
SxOrd: LL
SxRes: 0
TMC: 1
XIF: 1.0
XIR: 1.0
alphaqed: 0.007778240462411
alphas: 0.118
global_nx: 0
kbThr: 1.0 
kcThr: 1.0
ktThr: 1.0
mb: 4.92
mc: 1.51
mt: 172.5
nfref: 5
nf0: 4
fact_to_ren_scale_ratio: 1.0
```

[nlo_ok.zip](https://github.com/NNPDF/pineko/files/12831817/nlo_ok.zip)

[lo_fail.zip](https://github.com/NNPDF/pineko/files/12828697/lo_fail.zip)
[grid_lo.npy.zip](https://github.com/NNPDF/pineko/files/12829416/grid_lo.npy.zip)
[grid_nlo.npy.zip](https://github.com/NNPDF/pineko/files/12829417/grid_nlo.npy.zip)

### PR DESCRIPTION
**Pull request by scarlehoff**
Fixes #121 

(I haven't updated poetry.lock & stuff, but want the fktables I'm doing to "point" to a published commit)

### CODE REVIEW

File: pyproject.toml
Comment (AleCandido) [inline pyproject.toml:28]:
```suggestion
pineappl = "^0.6.2"
```
You do not need a pre-release any longer.

**Review by AleCandido — APPROVED** [review]
A very quick comment. It is fine to bump.

**Comment by AleCandido** [comment]
Ah, wait: you should run
```
poetry lock
```
and commit the resulting lock file.

(this is the reason why tests are failing, sorry to have noticed only after the approval)

File: pyproject.toml
Comment (felixhekhorn) [reply pyproject.toml]: 0d28bbc

**Pull request merged**
